framework/mesh/mesh.cc

```diff
@@ -28,21 +28,18 @@ CreateUnpartitioned1DOrthoMesh(std::vector<double>& vertices)
 
   // Create unpartitioned mesh
   auto umesh = std::make_shared<UnpartitionedMesh>();
-
-  umesh->GetMeshAttributes() = DIMENSION_1 | ORTHOGONAL;
+  umesh->SetDimension(1);
 
   // Create vertices
   size_t Nz = vertices.size();
 
-  umesh->GetMeshOptions().ortho_Nx = 1;
-  umesh->GetMeshOptions().ortho_Ny = 1;
-  umesh->GetMeshOptions().ortho_Nz = Nz - 1;
-  umesh->GetMeshOptions().boundary_id_map[4] = "ZMAX";
-  umesh->GetMeshOptions().boundary_id_map[5] = "ZMIN";
+  umesh->SetOrthoAttributes(1, 1, Nz - 1);
+  umesh->AddBoundary(4, "ZMAX");
+  umesh->AddBoundary(5, "ZMIN");
```

Comment:
Not suggesting to fix this now, but it has always pained me that max comes first in the integer ordering...

Reply:
I would personally change the boundary names to:
- left, right (1D)
- left, right, top, bottom (2D)
- left, right, top, bottom, front, back (3D)

Min/max has a different meaning to me. I don't associate it with boundary names...

Reply:
I was basing the left, right for 1D on the fact  that 1D would be in the X-coordinate. Maybe, I misunderstood, but I thought that was also a wanted change. Should 1D stay in the Z-coordinate?

Reply:
I would be totally fine with 1D being the x-coordinate, did we make that decision?